internal/brokers/testdata/TestIsAuthenticated/golden/error_when_authenticating

```diff
@@ -1,4 +1,4 @@
 FIRST CALL:
 	access: 
 	data: 
-	err: Broker "TestIsAuthenticated_Error_when_authenticating": IsAuthenticated errored out
+	err: Broker "TestIsAuthenticated": IsAuthenticated errored out
```

Comment:
nitpick: this commit should be just after [2cc0e16](https://github.com/ubuntu/authd/pull/38/commits/2cc0e1626b58223f0e4bdda0e818de9afbedf7be) for logical purposes

Reply:
Coming to think of it, they should actually be the same commit.

Since I have to rebase on main (due to some changes in the test files), I'll update this as well.